src/components/sidenav/sidenav-theme.scss

```diff
@@ -1,3 +1,5 @@
 md-sidenav.md-THEME_NAME-theme {
-  background-color: '{{background-color}}';
+  &, md-content {
+    background-color: '{{background-hue-1}}';    
```

Comment:
Okay, perfect :+1: By the way: #7374 fixes the basic theming issue with the sidenav.
So you PR will actually depend on that.